src/components/formInput/FormInput.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from "./formInput.module.css"
+
+export const FormInput = ({
+	label,
+	inputType = "text",
+	placeholder,
+	onChange,
+	isRequired = false,
+}: {
+	label: string
+	inputType?: string
+	placeholder: string
+	onChange: (e: any) => void
+	isRequired?: boolean
+}) => {
+	return (
+		<div className={styles.inputWrapper}>
+			<label className={styles.label} htmlFor={label}>
+				{label}
+			</label>
+			<input
+				className={styles.input}
+				type={inputType}
+				placeholder={placeholder}
+				onChange={onChange}
+				required={isRequired}
+			/>
```

Comment:
we should add an id prop to  `FormInput` that can then be used as the input's id and switch the label's prop as follows `htmlFor={id}`